openapi/api.yml

```diff
@@ -8,6 +8,184 @@ tags:
     description: Application store operations
 
 paths:
+  /applications/reporting/applications_statuses_data:
```

Comment:
@nuwan-samarasinghe Endpoints needed to be reordered to work correctly because of this new breaking change because of the connection 3 upgrade, "Route priority changed. The most specific route should now be defined first in the specification." https://connexion.readthedocs.io/en/latest/v3.html#smaller-breaking-changes